activities/staff-information/request-professional-development.md

```diff
@@ -0,0 +1,22 @@
+---
+type: Guide
+---
+
+# Professional Development
+
+Staff at the Foundation for Public Code is responsible for their professional development and growth.
+This means that staff needs to invest both in time and effort, such as courses, as a part of their job.
+This guide explains what you need to do in order to engage in learning opportunities.
+
+Professional development is fully paid for by the organization, both in time and learning costs,
```

Comment:
Is the line break in the end of this line intentional?

Reply:
Yes.

In order to keep tracking changes easy I'm trying to keep sentence fragments on separate lines so swaps and shifts can be better tracked through Git.

Reply:
Ok! But then you have missed quite a few sentence fragments. Should I mark them?

Reply:
I've looked through all of them again and think this is a good distinction there now.
I think the commas on the same line don't actually fragment the sentences here.

In the future, this guidance could be rewritten to be clearer and less spoken language to remove this confusion.
I suggest leaving it for now, however suggestions are always welcome.

Reply:
Okay, then your linguistic skills are way beyond mine and I hope I will not need to keep to the same standard in my pull requests :)